CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
+## [1.25.0] - 2021-11-15
+
+### Updated
+
+- [Documented known issue with tracking consent integrintegration when services set `platform.frontend.host` locally](docs/maintainers/journal/2021-11-15-tracking-consent-config-assumes-platform-tech-host-not-set-locally.md).
```

Comment:
Very trivial change but there is a typo "integrintegration"

Reply:
🤦‍♂️ thank you for catching that!